src/videojs.ads.js

```diff
@@ -25,6 +25,28 @@ var
       // another cancellation is already in flight, so do nothing
       return;
     }
+
+    // Avoid content flash on iOS
+    if (videojs.browser.IS_IOS) {
+
+      // A placeholder black box will be shown in the document while the player is hidden.
+      var placeholder = document.createElement('div');
+      placeholder.style.width = player.width() + 'px';
```

Comment:
Yes, `player.width` and `player.height` are both the *configured* values for the size of the player. They're match `videoEl.getAttribute('width')` which means if you don't have a size configured, you'll end up getting `0px` here.